components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m

```diff
@@ -370,6 +370,138 @@ - (void)testFiveActionsTooNarrowRTL {
   [self generateSnapshotAndVerifyForView:controller.view];
 }
 
+#pragma mark - contentEdgeInsets
```

Comment:
Can we make a new file for these tests and for the follow up tests `titleEdgeInsets` `imageEgdeInsets` so one file for 3 kinds of test `content/title/image edgeInsets`?

Reply:
Why do you feel the tests should go in another file? I thought it was nice not to duplicate all the helpers/config methods into another test file.